18F/identity-idp
# LG-14220 (1): Standardize completions screen logo

## 🎫 Ticket

Partially implements [LG-14220](https://cm-jira.usa.gov/browse/LG-14220)

## 🛠 Summary of changes

Updates consent screen to use the same heading image for both identity verified and IAL1 accounts.

## 📜 Testing Plan

Verify no regressions in the appearance of heading image for completions screen:

1. Have IdP and [sample application](https://github.com/18f/identity-oidc-sinatra) running simultaneously
2. Go to http://localhost:9292
3. Optional: Change "Level of service" option to "Identity-verified"
4. Click "Sign in"
5. Complete sign in process
6. When at the completions screen, observe that you see the logo as in the "After" screenshot below, regardless of whether you are sharing information for an identity-verified account

## 👀 Screenshots

Before|After
---|----
<img width="1075" alt="Screenshot 2024-08-20 at 4 10 53 PM" src="https://github.com/user-attachments/assets/b425a735-ffc3-495f-99b1-d85986b276d7">|<img width="1075" alt="Screenshot 2024-08-20 at 4 10 44 PM" src="https://github.com/user-attachments/assets/2105241f-62e3-4b5c-adb7-6d4ab01257fa">
